AppCenterDataStorage/AppCenterDataStorage/MSDataStore.m

```diff
@@ -261,6 +260,42 @@ - (void)createWithPartition:(NSString *)partition
                                }];
 }
 
+- (void)deleteDocumentWithPartition:(NSString *)partition
+                         documentId:(NSString *)documentId
+                       writeOptions:(MSWriteOptions *)__unused writeOptions
+                  completionHandler:(MSDataSourceErrorCompletionHandler)completionHandler {
+  // TODO consume writeOptions
+  [MSTokenExchange tokenAsync:(MSStorageIngestion *)self.ingestion
+                   partitions:@[ partition ]
+            completionHandler:^(MSTokensResponse *_Nonnull tokenResponses, NSError *_Nonnull error) {
+              // If error getting token.
+              if (error || !tokenResponses) {
+                MSLogError([MSDataStore logTag], @"Can't get CosmosDb token:%@", [error description]);
```

Comment:
Text can be improved like "Can't get CosmosDb token, error:%@"

Reply:
Use `error.localizedDescription`

Reply:
Message improved. Good to know about `error.localizedDescription`